static/app/views/dashboards/navigation.tsx

```diff
@@ -32,9 +42,30 @@ export default function DashboardsNavigation({children}: DashboardsNavigationPro
               {t('All')}
             </SecondaryNav.Item>
           </SecondaryNav.Section>
+          <SecondaryNav.Section title={t('Starred')}>
```

Comment:
do we still want to show the title even if there are no starred dashboards?

Reply:
For now yes. I think we will have an empty state at some point